package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "build": "yarn build:prod",
     "build:prod": "yarn clean && node --max_old_space_size=8192 node_modules/.bin/webpack --mode=production",
+    "check:updates": "yarn npm-check-updates",
```

Comment:
Does this need to be `npx npm-check-updates`?

Reply:
no, it's working with yarn
<img width="582" alt="image" src="https://user-images.githubusercontent.com/5322142/190201545-5ba4a57e-d356-47e2-9c7a-7f02ad6ce7fa.png">

Reply:
Ok, I mentioned it because we have other script targets that use `npx`